Chapters/FirstContact/FirstContactShrunk.md

```diff
@@ -98,7 +96,7 @@ Each application is able to define its own icon provider by defining a subclass
 
 #### Presenter interaction logic
 
-Now we should define what will happen when we press a button. 
+Now we define what will happen when the user presses a button. 
```

Comment:
We do not press the button. The user does. Or the reader. I would avoid using the "we" form.

Reply:
Agreed.